### ISSUE TITLE
NVDA 2013.3 does not read line by line in PowerPoint 2010 slide shows

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3759]
**Reported by baphomet on 2014-01-08 10:50**
In PowerPoint 2010 slide shows, NVDA 2013.3 does read the whole text from the current cursor position to the end of the current slide. This behavior is repeated whenever the current line is changed. However, I am able to read line by line after pressing CTRL+POS1/Home to get to the beginning of the current slide. With NVDA 2013.2, reading line by line was no problem.

**Comment by leonardder** [comment]
CC @MichaelDCurran

**Comment by ehollig** [comment]
I thought this was because Automatic Say All on page load was checked, but testing this in a slideshow after pressing F5, NVDA continues to read as if it is performing a skim reading say all.

**Comment by Adriani90** [comment]
@baphomet I cannot reproduce this. aFter pressing f5, NVDA reads everything. But when pressing up and down arrow I can read line by line as it is expected.
cc: @ehollig

**Comment by Adriani90** [comment]
@ehollig can you reproduce this? me not.

**Comment by ehollig** [comment]
I could be misunderstanding this issue, but I think the issue is that the entire powerpoint slide is read when Automatic Say All on page load is unchecked. If you want to read the first line of the next slide when in presentation mode, after pressing space to go to the next slide, the user must press control to stop speech prior to reading content on the slide line by line.

**Comment by Adriani90** [comment]
We do not have a confirmation from the author since more than 4 years. I am closing this as works for me. if someone has this problem please comment on this issue and we can reopen the discussion.
